go.mod

```diff
@@ -6,9 +6,9 @@ module github.com/jfrog/terraform-provider-project
 require (
 	github.com/go-resty/resty/v2 v2.7.0
 	github.com/hashicorp/terraform-plugin-docs v0.7.0
-	github.com/hashicorp/terraform-plugin-log v0.3.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.14.0
-	github.com/jfrog/terraform-provider-shared v0.5.0
+	github.com/hashicorp/terraform-plugin-log v0.4.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.16.0
+	github.com/jfrog/terraform-provider-shared v1.1.1
```

Comment:
Do tests pass with this version of shared? I am trying to track down these last 2 issues and it's not clear to me

Reply:
Yes, all passed